src/scenarios/billing/SubscribeToSubscriptionPlan/SubscribeToSubscriptionPlan.tsx

```diff
@@ -0,0 +1,476 @@
+            scenarios.
+          </p>
+          <p>
+            Billing Service should be configured with the following settings for
+            this scenario to work:
+          </p>
+          <p>
+            <b>Success Redirect URL:</b> {window.location.href}
+            /subscribe-success
+          </p>
+          <p>
+            <b>Error Redirect URL:</b> {window.location.href}/subscribe-error
+          </p>
+        </Container>
+
+        <Divider />
+
+        <Form>
+          <Form.Input
+            control={VariableSearch}
+            width={4}
+            icon="key"
+            label="User Access Token"
+            value={userAccessToken}
+            setStateValue={setUserAccessToken}
+          />
+
+          <Button
+            primary
+            onClick={async () => {
+              fetchPaymentPlans();
+            }}
+          >
+            Fetch Payment Plans & Paypal Settings
+          </Button>
+
+          <Divider />
+
+          <Form.Dropdown
+            fluid
+            selection
+            width={4}
+            label="Subscription Plans with PayPal Purchase"
+            placeholder="Select a subscription plan"
+            options={paymentPlans.map((plan) => {
+              return {
+                text: `[${plan.subscriptionPlan.title}] ${plan.title} (${
+                  plan.periodQuantity
+                } ${plan.periodUnit.toLocaleLowerCase()})`,
+                value: plan.id,
+              };
+            })}
+            value={paymentPlanId}
+            onChange={async (event, { value }) => {
+              const paymentPlanId = value as string;
+              setPaymentPlanId(paymentPlanId);
+              await fetchPaymentPlanPrices(paymentPlanId);
+            }}
+          ></Form.Dropdown>
+
+          <Form.Dropdown
+            fluid
+            selection
+            width={4}
+            label="Country Prices"
+            placeholder="Select an option"
+            options={paymentPlanPrices.map((option) => {
+              return {
+                text: `${option.country} [${option.price} ${option.currency}]`,
+                value: option.country,
+              };
+            })}
+            value={country}
+            onChange={(event, { value }) => {
+              setCountry(value as string);
+            }}
+          ></Form.Dropdown>
+
+          <Form.Field label="Purchase Flow"></Form.Field>
+          <Form.Radio
+            disabled={paypalInitializing}
+            label="Popup"
+            name="purchaseFlow"
+            checked={purchaseFlow === 'POPUP'}
+            onChange={() => changePurchaseFlow('POPUP')}
+          ></Form.Radio>
+          <Form.Radio
+            disabled={paypalInitializing}
+            label="Redirect"
+            name="purchaseFlow"
+            checked={purchaseFlow === 'REDIRECT'}
+            onChange={() => changePurchaseFlow('REDIRECT')}
+          ></Form.Radio>
```

Comment:
let's switch these buttons around - also that will help with resetting the selection into the "REDIRECT" option when the user has modified any of the parent dependencies (i.e. subscription plan / country) above

Reply:
After discussing this over a call, we decided to keep the radio buttons in the current order, and to clear the selection when needed instead of selecting "REDIRECT".